ppx-lib/gen_js_api_ppx.ml

```diff
@@ -385,7 +392,15 @@ let no_attributes attributes =
     ) attributes;
   attributes = []
 
-let rec parse_arg ~variance ctx lab ~global_attrs ty =
+type type_context =
+  {
+    type_params: label list;
+    packages: (label * label) list; (** reversed order, (local_name, module_name) *)
```

Comment:
Why `label` instead of `string` ? 
I know they are synonyms but it seems that label are only used in the context of polymorhic variants and objects.